core/src/oned/ODMultiUPCEANReader.cpp

```diff
@@ -62,46 +61,6 @@ static bool DecodeDigit(const PatternView& view, std::string& txt, int* lgPatter
 		AppendBit(*lgPattern, bestMatch >= 10);
 
 	return true;
-#else
```

Comment:
Please leave this disabled code alone. It might be useful as documentation in the future.